pipeline_documentation.md

```diff
@@ -165,14 +166,28 @@ Create index for [**STAR**](#third-party-software-used) short read aligner.
     - Chromosome name list `chrName.txt`; used in
       [**create_index_salmon**](#create_index_salmon)
 
+#### `sort_gtf`
+
+Sort provided gtf by chromosome, start and end coordinates. 
+
+> This process is executed once for the provided gtf annotation file.
+
+- **Input**
+  - Gene annotation file (`.gtf`)
+- **Output**
+  - Sorted Gene annotation file (`.gtf`); used in
```

Comment:
The sorted gtf is also used in `kallisto_merge_genes`, missing here, please include.